start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
+
+# Problem
+
+An InnerSource initiative is considered but not started because management is 
+unsure about its outcome and, as a result, is not willing to commit to an 
+investment.
+
+# Context
+
+The majority of software development projects in the company are distributed
+globally. The company is considering to leverage InnerSource to make the
+required collaboration more efficient. However, most managers are not familiar
+with the Open Source working model and are instead accustomed to hierarchical,
+top-down control style management. The idea of InnerSource is very popular with
+software developers in the company, not the least because many developers use
+or are actively developing Open Source software.
+
+# Forces
+
+- Managers will want to validate the claims of improved collaboration through
+  InnerSource before making a long term investment. This usually involves
+  putting numbers against the expected improvements, which are hard to come
+  by for InnerSource.
+- If the InnerSource initiative has a huge uptake among developers and if many
+  projects come to rely on it, a decision to shut it down will be very 
+  unpopular and therefore hard to make.
+- When implementing new working models such as InnerSource which are radically
+  different from working models previously practiced in the company, it is 
+  likely that existing, mandatory processes are no longer applicable and even
+  that one has to operate in a regulatory, sometimes legal no-mans land.
+
+# Solution
+
+Declare the InnerSource initiative as a time limited experiment. Define and
+communicate the criteria for evaluating the experiment and ensure that 
+sufficient data is gathered to perform the evaluation. Consider establishing
+a _Review Committee_ (**tbd**: pattern link) to increase the chances of 
+management buy-in.
+
+# Resulting Context
+
+Managers are able to kick start InnerSource for the following reasons:
+
+- The experimental setup eases the need to management to scrutinize their
+  numbers in the same way that they would for typical projects.
+- The possibility of failure of the experiment is understood and accepted. The
+  personal risk for the supporting managers is minimized.
+- Even in case of a failure, the setup ensures that the company will learn from
+  the experiment.
+- In case of success, the data gathered during the experiment will allow
+  managers to make a longer lasting commitment to InnerSource. 
+
+Participants in the InnerSource experiment are conscious of the fact that, as
```

Comment:
I don't understand these last two points in the resulting context - how are they good, desirable, or related to the problem or opening context?

Reply:
Hmm, you have a point there, @rrrutledge . What I wanted to express is that being transparent about the fact that the experiment might not continue would help avoid unnecessary frustration in the developer base. I can say that this wouldn't have helped me personally, though. One way in which the experimental nature of our InnerSource initiative did help was that it communicated to us developers that we had to prove that InnerSource (which we all love) does work well. Does that resonate with you, @rrrutledge ?

Reply:
I've rewritten this paragraph. Does this work better for you, @rrrutledge ?